fluent-bit-cloudwatch.go

```diff
@@ -26,12 +26,28 @@ import (
 
 	"github.com/sirupsen/logrus"
 )
-import "strings"
+import (
+	"log"
+	"strings"
+)
 
 var (
-	cloudwatchLogs *cloudwatch.OutputPlugin
+	pluginInstances []*cloudwatch.OutputPlugin
 )
 
+func addPluginInstance(ctx unsafe.Pointer, instance *cloudwatch.OutputPlugin) {
+	id := len(pluginInstances)
+	output.FLBPluginSetContext(ctx, id)
+	pluginInstances = append(pluginInstances, instance)
+}
+
+func getPluginInstance(ctx unsafe.Pointer) *cloudwatch.OutputPlugin {
+	id := output.FLBPluginGetContext(ctx).(int)
+	log.Printf("[cloudwatch] Flush called for id: %d", id)
```

Comment:
Do you think it would be more useful to log something like the `output.FLBPluginConfigKey(ctx, "endpoint")` instead of the index? I don't know if `"[cloudwatch] Flush called for id: 1"` would be useful to users

Reply:
Oh lol, this was just print line debugging... I think I'm not going to log anything here.

I think I might change the plugin to do the following for logging though:

Right now, when the plugin is initialized, it prints its config. I can add a statement that shows which plugin instance was initialized. And then, in all subsequent messages, the id can be present, so something like:

```
[cloudwatch] Initializing plugin instance 1
<print config>
[cloudwatch] Initializing plugin instance 2

...

[cloudwatch 1] Sent 400 events to CloudWatch
[cloudwatch 2] Error parsing log message: failed to marshal json
```